R/git.R

```diff
@@ -0,0 +1,22 @@
+#' Git information
+#' 
+#' Displays git information from the repository
+#' 
+#' @import git2r
+#' @import shiny
+#' @export
+git_info <- function() {
+  print(getwd())
+  repo <- init(path = ".", bare = FALSE)
```

Comment:
This is dangerous. Why do we initialize a git repo?
When I ran the example code in the console (shiny app was not stored in any file), my current directory was initialized as git repo. It should be rather "if my getwd() is a repo, then I display git info. Otherwise display that I am not in the repo"

Reply:
Yeah, I confused `init` functionality with `repository`. Fixed.